packages/cryptoassets/src/chains/mainnet/evm/avalanche.ts

```diff
@@ -0,0 +1,65 @@
+import { EvmChain } from '../../../chains/EvmChain';
+import { AssetTypes, ChainId } from '../../../types';
+
+export default new EvmChain({
+  id: ChainId.Avalanche,
+  name: 'Avalanche',
+  code: 'AVALANCHE',
+  color: '#E84141',
+
+  nativeAsset: [
+    {
+      name: 'Avalanche',
+      chain: ChainId.Avalanche,
+      type: AssetTypes.native,
+      code: 'AVAX',
+      priceSource: { coinGeckoId: 'avalanche-2' },
+      color: '#E84141',
+      decimals: 18,
+    },
+  ],
+
+  isEVM: true,
+  hasTokens: true,
+  isMultiLayered: false,
+
+  averageBlockTime: 3,
+  safeConfirmations: 10,
+  txFailureTimeoutMs: 600_000,
+
+  network: {
+    name: 'avalanche_mainnet',
+    coinType: '60',
+    networkId: 43114,
+    chainId: 43114,
+    isTestnet: false,
+    rpcUrls: [
+      'https://speedy-nodes-nyc.moralis.io/7c28a10f7d39bfb24704dafc/avalanche/mainnet',
```

Comment:
Is it possible to read moralis key from `.env`

Reply:
no, cryptoassets cannot have env variables